skipper.env

```diff
@@ -40,3 +40,4 @@ MINIKUBE_HOME
 OPENSHIFT_VERSION
 ENABLE_ORG_TENANCY
 ENABLE_ORG_BASED_FEATURE_GATES
+TEST
```

Comment:
what is that for?

Reply:
>This will allow specifying a package to test

It's in the commit message :wink: 

It was easier for me to specify the package and then focus the tests I wanted in the code rather than using the focus regex on the command line when running tests.

Reply:
i'm reviewing everything together so i looked at the description not the commit message, thanks